website/project/model.py

```diff
@@ -2535,12 +2535,14 @@ def add_unregistered_contributor(self, fullname, email, auth,
         self.save()
         return contributor
 
-    def set_privacy(self, permissions, auth=None, log=True, save=True):
+    def set_privacy(self, permissions, auth=None, log=True, save=True, skip_mail=False):
         """Set the permissions for this node.
```

Comment:
Also, please update the docstring, since now `set_privacy` also triggers the `new_public_project_emails`